### PR TITLE
Remove expectations from the test suite in favor of TestSchedulers.

### PR DESCRIPTION
**Pull request by jasdev**
This PR pays down some test debt we’ve been sitting on without a `TestScheduler` type — and thankfully, the Point-Free folks open-sourced [theirs](https://github.com/pointfreeco/combine-schedulers)! The test suite went from running in roughly ~ten seconds locally (on a maxed-out ’19 MacBook Air) to under half a second.

Screen Shot 2020-06-15 at 9 43 11 PM

### CODE REVIEW

**Comment by jasdev** [comment]
hmm — actually this would require us to bump CombineExt’s deployment targets considerably (from looking at the CI failures).

**Comment by jasdev** [comment]
(closing for now until we find a way forward on the version targeting front — @mbrandonw mentioned he might be up for doing a similar `@availability` fencing in the schedulers repo.)

**Comment by mbrandonw** [comment]
We're def down to having it added, but our schedules are a bit tight for the next week or more. If you've got time to PR it we'll happily take it, or we can sync up again sometime soon!

**Comment by freak4pc** [comment]
I've made a PR over here: https://github.com/pointfreeco/combine-schedulers/pull/3

I switched Package.swift to my local branch and everything seems to run:
```swift
.package(url: "https://github.com/freak4pc/combine-schedulers", .branch("relax-deployment-target")),
```

**Comment by jasdev** [comment]
(once CI is clear, should be good for a pass over @freak4pc — thank you for taking care of the versioning snag!)

File: Sources/Operators/PrefixDuration.swift [hunk 1]
```diff
@@ -7,30 +7,39 @@
 //
 
 #if canImport(Combine)
+#if canImport(CombineSchedulers)
```
Comment (freak4pc) [inline Sources/Operators/PrefixDuration.swift:10]: dont think this line is needed

File: Tests/CreateTests.swift [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import XCTest
 import Combine
 import CombineExt
+import CombineSchedulers
```
Comment (freak4pc) [inline Tests/CreateTests.swift:13]: do you even use the test scheduler here or did you just remove expecations?

File: Tests/FlatMapLatestTests.swift [hunk 3]
```diff
@@ -9,7 +9,8 @@
 #if !os(watchOS)
 import XCTest
 import Combine
-import CombineExt
+@testable import CombineExt
```
Comment (freak4pc) [inline Tests/FlatMapLatestTests.swift:12]: Why `@testable` is needed ?

Comment (jasdev) [reply Tests/FlatMapLatestTests.swift:12]: Great catch, this is a leftover change left from an intermediary commit that I later squashed.

File: Tests/CreateTests.swift (hunk 2 above)
Comment (jasdev) [reply Tests/CreateTests.swift]: Another lingering import, thanks for scouting it!

**Pull request merged**